apps/src/templates/sectionProgress/progressTables/ProgressTableContentView.jsx

```diff
@@ -68,7 +68,8 @@ export default class ProgressTableContentView extends React.Component {
     const lesson = this.props.scriptData.stages[columnIndex];
     const includeArrow =
       this.props.includeHeaderArrows &&
-      (lesson.levels.length > 1 || lesson.levels[0].isUnplugged);
+      (!!lesson.levels?.length &&
```

Comment:
When a lesson has no levels, is `lesson.levels` undefined or is it an empty array?

I see this check (or similar) repeated a couple of times and the intent of the check isn't immediately clear.  Perhaps we can add a utility method somewhere to determine if a lesson has no levels?